uber/templates/summary/csv_import.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base-admin.html" %}
+{% block title %}CSV Import{% endblock %}
+{% block content %}
+
+<h2> CSV Table Imports </h2>
+This attempts to import a csv file for the table you select. <strong>Please do not refresh this page.</strong>
```

Comment:
This is the main thing I'm still unhappy about. If you refresh the import page, it will re-submit the form and re-import everything.

The typical way around this is by using HTTPRedirect to clear the POST data, but we _use_ the POST data to display what the admin just uploaded. Trying to pass the data back through a GET request munges it.